packages/jest-runtime/src/index.ts

```diff
@@ -150,6 +150,7 @@ class Runtime {
     | null;
   private _internalModuleRegistry: ModuleRegistry;
   private _isCurrentlyExecutingManualMock: string | null;
+  private _mainModule: Module | null;
```

Comment:
make sure to set this to `null` in `teardown`

Reply:
Done.